generate-renode-scripts.py

```diff
@@ -68,37 +67,31 @@ def generate_registration_entry(address, size=None, name=None):
     return result
 
 
-def generate_ethmac(peripheral, shadow_base, **kwargs):
+def generate_ethmac(peripheral, **kwargs):
     """ Generates definition of 'ethmac' peripheral.
 
     Args:
         peripheral (dict): peripheral description
-        shadow_base (int or None): shadow base address
         kwargs (dict): additional parameters, including 'buffer'
 
     Returns:
         string: repl definition of the peripheral
     """
     buf = kwargs['buffer']()
     phy = kwargs['phy']()
+    phy['size'] = 0x800
```

Comment:
Should probably be a `FIXME: Get litex to generate the phy buffer memory region into output information`

Reply:
`ethphy` is a CSR (not a memory region) so I rephrased the comment a little